tests/Microsoft.Bot.Builder.AI.LanguageGeneration.Tests/LanguageGenerationApplicationTests.cs

```diff
@@ -7,21 +7,75 @@ namespace Microsoft.Bot.Builder.AI.LanguageGeneration.Tests
     [TestClass]
     public class LanguageGenerationApplicationTests
     {
-        private const string Endpoint = "https://platform.bing.com/speechdx/lg-dev/v1/lg";
+        private readonly string applicationId = "test_application";
+        private readonly string applicationRegion = "westus";
```

Comment:
same

Reply:
These are already constants, or you mean something different?